storage/mysql/rootStore.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS Roots(
+  Id BINARY(32),
+  DER MEDIUMBLOB,
+  PRIMARY KEY(Id)
+);
+
+CREATE TABLE IF NOT EXISTS RootSets(
+  RootSetID Binary(32),
+  RootID Binary(32),
+  PRIMARY KEY(RootSetID, RootID)
+);
+
+CREATE INDEX RootSetIDIndex ON RootSets(RootSetID);
+CREATE INDEX RootIDIndex ON RootSets(RootID);
```

Comment:
When do you envisage querying only by RootID?

Reply:
Removed